modules/oe_content_entity/modules/oe_content_entity_venue/config/install/core.entity_view_display.oe_venue.oe_default.default.yml

```diff
@@ -13,6 +13,24 @@ targetEntityType: oe_venue
 bundle: oe_default
 mode: default
 content:
+  created:
```

Comment:
is this here after a config devel export? asking because I dont see the relation to the ticket

Reply:
Yes, it is default state of the configuration after fresh Drupal installation.